src/organization/organization.selectors.js

```diff
@@ -0,0 +1,47 @@
+import { createSelector } from 'reselect';
+import { getOrganization as getOrganizationFromStore } from '../../root.reducer';
+
+export const getOrganization = createSelector(
```

Comment:
Benefits of using `reselect` instead of querying the store `by.doing.something.like.this`

```
Selectors can compute derived data, allowing Redux to store the minimal possible state.
Selectors are efficient. A selector is not recomputed unless one of its arguments change.
Selectors are composable. They can be used as input to other selectors.
```
More info https://github.com/reactjs/reselect